### DISCUSSION TITLE
Nice But . . .

### DISCUSSION THREAD

**Discussion by corinthian13** [willdoescode/nat#63] [General]
The **natls** command is a visual improvement on bare **ls**.
But no package that installs by default to only run on root (admin) account deserves to be used.
Period.

No use saying we can all just config it to run in a directory accessible by all users.
We could.
But so could you.
Who the hell only works on root account anyway ?
And this package is intended to speed up _work_, isn't it ?

**Comment by willdoescode** [comment]
I don't control what steps the package manager uses to install I just upload my project.

**Comment by corinthian13** [comment]
Yes and no. 
It's caused by your app being locked within the .cargo folder - which for some crazy reason is kept in the admin's home folder.
Any way you could keep your app in a separate folder from the cargo thing ?

**Comment by willdoescode** [comment]
it should not be an issue with .cargo as brew is able to install it ok. All rust installs go to .cargo and if you want the binary somewhere else than you can download the most recent binary from releases and move it to your preferred location.

**Reply by corinthian13** [reply]
My PC now says that neither natls nor cargo is installed with either apt or snap on my system. Yet my CLI still responds to natls -l as if nat were still there. I can't use it on user accounts nor can I remove it altogether. 
Classic panties tangle. All down to you entrusting your software to the care of this cargo outfit who provide a 'free' platform for coders doing apps in Rust. 
Yeah it's free for you - at our expense.

**Reply by willdoescode** [reply]
Cargo is made by the creators of rust and open source, im not sure what you are getting at.
